build-system/compile/bundles.config.bento.json

```diff
@@ -4,7 +4,6 @@
     "version": "1.0",
     "latestVersion": "0.1",
     "options": {
```

Comment:
For the purposes of Bento configuration json, does it make sense to remove `latestVersion`, and the entire options obj? `npm` and `bento` will always be true

Reply:
Yeah, that makes sense. I'll remove it from the config but stub it in JS for now because there are shared parts of the build process that check those options.